src/public-request/transaction-prisma.service.ts

```diff
@@ -0,0 +1,110 @@
+import { Injectable } from "@nestjs/common";
+import { PrismaService } from "@app/prisma/prisma.service";
+import type { PublicRequests as PublicRequestModel, Prisma } from "@prisma/client";
+
+@Injectable()
+export class TransactionPrismaService {
```

Comment:
Cambiar nombre del servicio a:
```suggestion
export class PublicRequestPrismaRepository {
```